cogs/cryochamber.py

```diff
@@ -0,0 +1,72 @@
+import re
+
+from discord import Embed, Colour
+from discord.ext.commands import Cog, CommandError, MessageConverter, TextChannelConverter, group
+
+
+class Cryochamber(Cog):
+    def __init__(self, _):
+        self.ongoing_preservations = set()
+
+    @group(invoke_without_command=True)
+    async def preserve(self, context, *, arg):
```

Comment:
Since this is the master command of the `preserve` group, any documentation given as the docstring here will show as the main help of that group (see the tiles cog source, or say `?help tile` to the current Cryptogull. This would help with testing too since we can have examples of how it's supposed to be used.